Makefile

```diff
@@ -138,6 +138,15 @@ e2e-tests-self-provisioned-es-kafka: prepare-e2e-tests deploy-kafka-operator dep
 	@echo Running Self provisioned Elasticsearch and Kafka end-to-end tests...
 	@STORAGE_NAMESPACE=$(STORAGE_NAMESPACE) ES_OPERATOR_NAMESPACE=$(ES_OPERATOR_NAMESPACE) ES_OPERATOR_IMAGE=$(ES_OPERATOR_IMAGE) go test -tags=self_provisioned_elasticsearch_kafka ./test/e2e/... $(TEST_OPTIONS)
 
+.PHONY: prepare-e2e-tests-kafka-kerberos
+prepare-e2e-tests-kafka-kerberos: es
+	@kubectl create -n default -f ./test/kafka-kerberos.yml
+
+.PHONY: e2e-tests-kafka-kerberos-es
```

Comment:
Should `e2e-tests-kafka-kerberos-es` be added to this target?
https://github.com/jaegertracing/jaeger-operator/blob/584df0b16b0f75058874179ac7c29ccd9fb2a370/Makefile#L90

Reply:
Yes, it should be, I updated the kafka with kerberos image some time ago, wondering if I break something. I'll review tomorrow to see if that is an issue. Thanks!